### PR TITLE
added translation to users

### PR DESCRIPTION
**Pull request by Andrii-Kozak**
[Trello Task](https://trello.com/_____)

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [x] screenshot of feature is added if task has UI

### CODE REVIEW

**Review by Swooty18 — APPROVED** [review]

**Review by pereslop — APPROVED** [review]

**Review by mmarusyk — APPROVED** [review]

File: app/javascript/stylesheets/users.scss [hunk 1]
```diff
@@ -65,20 +72,25 @@ span.insert-user {
...
 }
 
-.show-box{
+.navbar-user {
+  margin-top: 20px;
+}
+
+.show-box {
   display: flex;
   flex-direction: row-reverse;
```
Comment (pereslop) [inline app/javascript/stylesheets/users.scss:95]: why here row-rewerse opition? in row users is sorted in reverse from right to left?

File: app/views/account/users/edit.html.slim [hunk 2]
```diff
@@ -1,7 +1,7 @@
-- provide(:create_btn_text, 'Edit User')
-- provide(:cancel_btn_text, 'Cancel')
-- provide(:path, account_users_path)
+- provide(:create_btn_text, t('.btn-edit'))
+- provide(:cancel_btn_text, t('.btn-cancle'))
```
Comment (pereslop) [inline app/views/account/users/edit.html.slim:2]: what mean `btn-cancle?` typo

File: config/locales/en.yml [hunk 3]
```diff
@@ -119,12 +119,50 @@ en:
...
+      form:
+        sex:
+          male: 'Male'
+          female: 'Female'
+          no_sex: 'not set'
+        first-name: 'First Name'
+        placeholder-first-name: 'Enter first name'
+        last-name: 'Last Name'
+        placeholder-last-name: 'Enter last name'
+        user-birth: 'Date of Birth: '
```
Comment (pereslop) [inline config/locales/en.yml:151]: do you need here extra space?

File: config/locales/ua.yml [hunk 4]
```diff
@@ -1,4 +1,14 @@
 ua:
+  en:
+  views:
+    pagination:
```
Comment (pereslop) [inline config/locales/ua.yml:4]: good

File: config/locales/en.yml (hunk 3 above)
Comment (Andrii-Kozak) [reply config/locales/en.yml:151]: image

File: app/javascript/stylesheets/users.scss (hunk 1 above)
Comment (Andrii-Kozak) [reply app/javascript/stylesheets/users.scss]: fixed

File: app/views/account/users/edit.html.slim (hunk 2 above)
Comment (Andrii-Kozak) [reply app/views/account/users/edit.html.slim]: fixed

File: config/locales/en.yml (hunk 3 above)
Comment (Andrii-Kozak) [reply config/locales/en.yml]: fixed

**Review by chu4 — APPROVED** [review]

**Pull request merged**